dataproc_jupyter_plugin/controllers/dagController.py

```diff
@@ -15,7 +15,7 @@
 
 import json
 import subprocess
-from dataproc_jupyter_plugin.commons.gcloudOperations import GetCachedCredentials
+from dataproc_jupyter_plugin import credentials
 from jupyter_server.base.handlers import APIHandler
 import tornado
 from dataproc_jupyter_plugin.services.dagListService import (
```

Comment:
The `dagListService.py`, `dagRunService.py`, `importErrorService.py`, and `triggerDagService.py` files need to all be combined into a single file under the `dataproc_jupyter_plugin.services` directory named `airflow.py`, which follows the same pattern used in the [bigquery package](https://github.com/GoogleCloudDataproc/dataproc-jupyter-plugin/blob/main/dataproc_jupyter_plugin/services/bigquery.py).

That file should have a single class in it named `Client` with the following methods:

* `list_dags`
* `delete_dag`
* `update_dag`
* `list_dag_runs`
* `list_dag_run_tasks`
* `list_dag_run_task_logs`
* `list_import_errors`
* `trigger_dag`

That class should take in its constructor the credentials and logger used by all of those methods, and if the credentials are not sufficiently populated, an error should be raised in the constructor.

This import line will then become `from dataproc_jupyter_plugin.services import airflow`.